frontend/catalyst/utils/runtime_environment.py

```diff
@@ -30,9 +30,18 @@
     "oqc_runtime": os.path.join(package_root, "../../catalyst/third_party/oqc/src/build"),
 }
 
+DEFAULT_BIN_PATHS = {
+    "cli": os.path.join(package_root, "../../../mlir/build"),
+}
 
 def get_lib_path(project, env_var):
     """Get the library path."""
     if INSTALLED:
         return os.path.join(package_root, "..", "lib")  # pragma: no cover
     return os.getenv(env_var, DEFAULT_LIB_PATHS.get(project, ""))
+
+def get_bin_path(project, env_var):
+    """Get the library path."""
+    if INSTALLED:
+        return os.path.join(package_root, "..")  # pragma: no cover
```

Comment:
I think we should move this into an actual `bin` folder.

```suggestion
        return os.path.join(package_root, "..", "bin")  # pragma: no cover
```